src/controllers/softtakeover.cpp

```diff
@@ -104,29 +103,29 @@ bool SoftTakeover::ignore(ControlObject* control, double newParameter) {
      *          (regardless of the new value's arrival time)
      *      - the new value arrives awhile after the previous one
      *          (regardless of what the previous value was)
-     * 
+     *
      * Sheesh, this is much easier to show in a truth table!
-     * 
+     *
      * Sides    prev distance   new distance    new value arrives   Ignore
      * opposite close           far             later               TRUE
      * opposite far             far             later               TRUE
      * same     close           far             later               TRUE
      * same     far             far             soon                TRUE
      * same     far             far             later               TRUE
-     * 
+     *
      *      Don't ignore in every other case.
      */
 
-    qint64 currentTime = Time::elapsedMsecs();
+    mixxx::Duration currentTime = Time::elapsed();
     // We will get a sudden jump if we don't ignore the first value.
-    if (m_time == 0) {
+    if (m_time == mixxx::Duration::fromMillis(0)) {
```

Comment:
Yea -- Duration(0) is shorter but doesn't say anything about units and would allow you to write Duration(5) as well which would make it easier to accidentally make mistakes with units.